tools/check-ffmpeg.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+have_ffmpeg=1
```

Comment:
I would put this code into a function and `source` this file, then call the function in the other scripts. Also move the last clause (with `exit 1`) back into the original scripts so that you don't need to have `-e` active.